src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -450,3 +450,192 @@ HttpStatusCode processRegisterContext
 
```

Comment:
MongoCommonRegister.cpp is focused in NGSIv1. It would be better to use a new file, to keep NGSIv2 code separated.

For instance, MongoRegisterExtractors.cpp or MongoRegisterSetters.cpp

Reply:
No worries, the entire set of functions will be hidden in a single cpp file, for now ...

Reply:
Fixed in 33bfe33